cluster/testdata/tls_config_node1.yml

```diff
@@ -1,5 +1,9 @@
 tls_server_config:
-  cert_file: "server.crt"
-  key_file: "server.key"
-  client_ca_file: "tls-ca-chain.pem"
+  cert_file: "certs/node1.pem"
+  key_file: "certs/node1-key.pem"
+  client_ca_file: "certs/ca.pem"
   client_auth_type: "VerifyClientCertIfGiven"
+tls_client_config:
```

Comment:
@brian-brazil is this the sort of config that [you suggested I add](https://github.com/prometheus/alertmanager/pull/2237#issuecomment-758888894)? Or was it something else? Thanks!